pyinterprod/interpro/iprscan.py

```diff
@@ -697,6 +549,7 @@ def _update_partition(uri: str, table: str, partitioned_table: str,
 
         if subparts:
             """
+            # the temporary table = staging table (TMP_TABLE?)
             The target table is sub-partitioned: the staging table needs
             to be partitioned
```

Comment:
Yes, staging = temporary. Better use _temporary_ to avoid further confusion.

```suggestion
            The target table is sub-partitioned: the temporary table needs
            to be partitioned
```